lm_eval/tasks/crows_pairs/crows_pairs_english.yaml

```diff
@@ -18,4 +18,4 @@ metric_list:
     higher_is_better: false
   - metric: acc
     aggregation: mean
-    higher_is_better: true
+    higher_is_better: false
```

Comment:
For crowspairs it's better to have a lower accuracy?

Reply:
this metric is meant to be named “pct_stereotype”, sorry! Thought I changed it, guess not